.env

```diff
@@ -128,4 +128,6 @@ LLM_SUMMERIZATION=true
 # PUBLIC_APP_DATA_SHARING=1
 # PUBLIC_APP_DISCLAIMER=1
 
-ENABLE_ASSISTANTS=false #set to true to enable assistants feature
\ No newline at end of file
+ENABLE_ASSISTANTS=false #set to true to enable assistants feature
+ASSISTANTS_GENERATE_AVATAR=true #requires an hf token, uses the model description and name to generate an avatar using a text to image model
+TEXT_TO_IMAGE_MODEL="runwayml/stable-diffusion-v1-5"
```

Comment:
nit: but we can use newer models like https://huggingface.co/latent-consistency/lcm-lora-ssd-1b (assuming that they are hosted on inference API) that produce higher quality images faster

1. It is higher quality because, it is based on SD XL
2. It produces images faster, because it is smaller model (ssd) & uses fewer sampling (lcm-lora)

Reply:
Ah that's fair! I picked sd1.5 over sdxl because I assumed it'd be faster but if we have faster models available by all mean let's change it :grin: 

I think we should prio speed>image quality for this feature, most of the time this will be seen as only a small thumbnail, so if you have good model recommendations, feel free!

Reply:
for example, https://huggingface.co/latent-consistency/lcm-lora-sdv1-5 would be faster than `runway/sd-1.5`. https://huggingface.co/latent-consistency/lcm-lora-sdxl might still be faster than `runway/sd-1.5`. Maybe @patil-suraj @sayakpaul can confirm?

Reply:
https://huggingface.co/blog/lcm_lora

Reply:
Since it's SDXL, it will not be SD speed. You can try out Segmind's SSD-1B.

Reply:
So https://huggingface.co/latent-consistency/lcm-lora-ssd-1b  would be the best, in terms of speed/qualiy tradeoff?

Reply:
I would say so. But, would also consider playing SD Turbo.

Reply:
What's the license of SD Turbo ?

Reply:
I tried `latent-consistency/lcm-lora-ssd-1bz` but it doesn't seem to load, I think it would need to be pinned in the API on our side if we go for it, just a note not necessarily a blocker 😁